docs/changing-an-existing-content-schema.md

```diff
@@ -0,0 +1,48 @@
+# Changing an existing content schema
+
+## General workflow
+
+* create a branch
+* make changes to the relevant schema fragment under `formats/<format name>.jsonnet`
+* change the relevant frontend examples (or add a new example)
+* run `rake` to compile a new schema.json and test the example against the schema
+* commit and push
+* open a PR
+* watch the multi-build status to see if all apps are compatible with the change
+* if the builds are successful, it can be merged
+
+## Adding a new field to a content schema
+
+Follow the General workflow described above. You are now free to add support for the field to apps. If you are adding a mandatory field you will need to additionally:
+
+* deploy the publishing app with changes to always populate the field
+* populate the field in any records inside content-store
+* follow the General workflow, adding the field to the `required` attribute
+* you can then update the frontend app to expect the field to always be present
+
+For example, imagine that you need to add a new optional field to the details hash of the `case_study` schema. The steps would be:
+
+1. edit [`formats/case_study.jsonnet`](/formats/case_study.jsonnet) to
+   add the new optional field
+1. run `rake`. This will:
+  1. regenerate the publisher [`schema.json`](/dist/formats/case_study/publisher/schema.json) to incorporate the changes you made to the `details`
```

Comment:
Style issue: these show up as part of the top-level list when I preview. Maybe it's just my local previewer, but it might be worth adding a newline here. Also worth noting this section uses numbers, whereas others use bullets.

On the other hand, I don't think we need to say all this here. If anything it should be in the README section about rebuilding schemas.

Reply:
It works for me in my markdown previewer, so assume it will be fine 👍